registry/insecure.md

```diff
@@ -5,43 +5,46 @@ title: Test an insecure registry
 ---
 
 While it's highly recommended to secure your registry using a TLS certificate
-issued by a known CA, you may alternatively decide to use self-signed
-certificates, or even use your registry over plain http.
-
-You have to understand the downsides in doing so, and the extra burden in
-configuration.
+issued by a known CA, you can choose to use self-signed certificates, or use
+your registry over an unencrypted HTTP connection. Either of these choices
+involves security trade-offs and additional configuration steps.
 
 ## Deploying a plain HTTP registry
 
 > **Warning**:
 > it's not possible to use an insecure registry with basic authentication.
 {:.warning}
 
-This basically tells Docker to entirely disregard security for your registry.
-While this is relatively easy to configure the daemon in this way, it is
-**very** insecure. It does expose your registry to trivial MITM. Only use this
-solution for isolated testing or in a tightly controlled, air-gapped
-environment.
+This procedure configures Docker to entirely disregard security for your
+registry. This is **very** insecure and is not recommended. It exposes your
+registry to trivial man-in-the-middle (MITM) attacks. Only use this solution for
+isolated testing or in a tightly controlled, air-gapped environment.
 
-1. Open the `/etc/default/docker` file or `/etc/sysconfig/docker` for editing.
+1.  Edit the `daemon.json` file, whose default location is
+    `/etc/docker/daemon.json` on Linux. If the `daemon.json` file does not exist,
+    create it. Assuming there are no other settings in the file, it should have
+    the following contents:
 
-    Depending on your operating system, your Engine daemon start options.
+    ```json
+    {
+      "insecure-registries" : ["myregistrydomain.com:5000"]
```

Comment:
Yeah I do tell them how to get there but didn't want to go into all the details.